src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -38,14 +38,16 @@ const OrgProfileTab: FC = () => {
 
   const dispatch = useDispatch()
 
+  const identityOrgId = identity.currentIdentity.org.id
```

Comment:
identity.currentIdentity.org.id all have proper default values, so there is no need for optional chaining here, since the default value of `identity.currentIdentity.org.id` is `''`